### PR TITLE
Make installable by `pip install git+`

### PR DESCRIPTION
**Pull request by Hoxbro**
I wanted to try out the module with: `python -m pip install git+https://github.com/martindurant/pandas-decimal`, but got the following error.  This is fixed by adding `[tool.setuptools.packages.find]` to `pyproject.toml`. 

Also, I changed flake8 to the github repo and ran `pre-commit run --all-files`. 

``` python-traceback
❯ python -m pip install git+https://github.com/martindurant/pandas-decimal
Collecting git+https://github.com/martindurant/pandas-decimal
  Cloning https://github.com/martindurant/pandas-decimal to /tmp/pip-req-build-79x1j4gq
  Running command git clone --filter=blob:none --quiet https://github.com/martindurant/pandas-decimal /tmp/pip-req-build-79x1j4gq
  Resolved https://github.com/martindurant/pandas-decimal to commit 8b1bcd8957f9a80441b373b957505be32cb464c8
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      /tmp/pip-build-env-lzwhwyiv/overlay/lib/python3.10/site-packages/setuptools/config/expand.py:144: UserWarning: File '/tmp/pip-req-build-79x1j4gq/LICENCE' cannot be found
        warnings.warn(f"File {path!r} cannot be found")
      error: Multiple top-level packages discovered in a flat-layout: ['notebooks', 'pandas_decimal'].
      
      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.
      
      If you are trying to create a single distribution with multiple packages
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:
      
      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names
      
      To find more information, look for "package discovery" on setuptools docs.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

### CODE REVIEW

**Comment by Hoxbro** [comment]
Found some small bugs, which I also added to this PR. I can also split it into new PRs if you want me to do that.

File: pandas_decimal/array.py [hunk 1]
```diff
@@ -180,7 +180,9 @@ def take(self, indices, *, allow_fill=False, fill_value=None):
         return self[indices]
 
     def copy(self):
-        return type(self)(np.copy(self._data), dtype=self._dtype)
+        return type(self)(
+            np.copy(self._data / 10**self.dtype.decimal_places), dtype=self._dtype
```
Comment (martindurant) [inline pandas_decimal/array.py:184]: This one looks wrong, shouldn't the new extension array have exactly the same data as the original? If it corrects the data but keeps the same number of decimal places in the dtype, you end up correcting twice.

**Review by martindurant — COMMENTED** [review]
Thanks, and congratulations for being the first external contributor.

I have absolutely no problem fixing multiple things in one PR at this very early stage of the package.

I have just the one comment for you.

File: pandas_decimal/array.py (hunk 1 above)
Comment (martindurant) [reply pandas_decimal/array.py:184]: ah right, should have been
```
return type(self).from_internal(np.copy(self._data), dtype=self._dtype)
```
(this involves no compute)

Comment (Hoxbro) [reply pandas_decimal/array.py]: This is now added.

**Pull request merged**